.github/workflows/makefile_ci.yml

```diff
@@ -21,8 +21,8 @@ jobs:
       - name: Install dependencies
         run: make install
 
-      - name: Run tests
-        run: make test
+      - name: Run CLI tests
+        run: make test_cli
```

Comment:
We prefer to use test-cli instead of text_cli

Reply:
Done